tests/apis/test_export.py

```diff
@@ -35,6 +36,26 @@ def test_export_moco(self):
         if stat != 0:
             print(output)
 
+    def test_export_yolox(self):
+        config_file = 'configs/detection/yolox/yolox_s_8xb16_300e_voc.py'
+        ori_ckpt = PRETRAINED_MODEL_YOLOXS
+        ckpt_path = f'{self.tmp_dir}/export_yolox_s_epoch300.pt'
+        stat, output = subprocess.getstatusoutput(
+            f'python tools/export.py {config_file} {ori_ckpt} {ckpt_path}')
+        self.assertTrue(stat == 0, 'export model failed')
+        if stat != 0:
+            print(output)
+
+    def test_export_yolox_jit(self):
```

Comment:
All cases should be tested, including end2end、 without end2end、with blade、raw and so on